TCAT/Supporting Files/Constants.swift

```diff
@@ -237,6 +238,13 @@ struct Constants {
         static let maxZoom: Float = 25
         static let searchRadius = 24140
     }
+    
+    struct Notification {
+        static let notifyBeforeBoarding = "Notify me 10 min before boarding"
+        static let notifyDelay = "Notify me about delays"
+        static let beforeBoardingConfirmation = "You will receive notifications 10 min before boarding time"
+        static let delayConfirmation = "You will receive notifications for delays in"
```

Comment:
Are lines 245-6 used anywhere?

Reply:
They aren't, but they'll be used for the in-app notification banners later

Reply:
Ok gotcha